Documentation/prerequisites-for-building.md

```diff
@@ -50,3 +50,47 @@ sudo zypper install cmake libuuid-devel icu libcurl-devel zlib-devel
 Make sure you run with Ubuntu 16.04 Xenial userland (this is the default after Windows 10 Creators Update, but if you enabled the "Bash on Ubuntu on Windows" feature before the Creators Update, you need to [upgrade manually](https://blogs.msdn.microsoft.com/commandline/2017/04/11/windows-10-creators-update-whats-new-in-bashwsl-windows-console/)). Running `lsb_release -a` will give you the version.
 
 Then follow the Ubuntu 16.04 instructions above.
+
+# clang 3.9 not found on Linux
+
+If you encounter this error, CoreRT could not find the clang executable
+```
+error : Platform linker ('clang-3.9') not found. Try installing clang-3.9 or the appropriate package for your platform to resolve the problem.
+```
+
+CoreRT expect by default clang-3.9 installed (see above). You can override this by setting an environment variable.
+
+```
+export CppCompilerAndLinker=clang
+```
+
+This works for building CoreCR itself as well as building with CoreRT.
+When filing bugs, please make sure to mention the clang version you are using, if you override this.
+
+# libObjWriter.so not found on Linux
+
+```
+EXEC : error : Unable to load shared library 'objwriter' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: libobjwriter: cannot open shared object file: No such file or directory 
+```
+
+This is the default error message when a [DLLImport] could not be loaded. CoreRT nuget packages distribute this file, but it might be failing to load depencies.
```

Comment:
```suggestion
This is the default error message when a `[DllImport]` could not be loaded. CoreRT nuget packages distribute this file, but it might be failing to load dependencies.
```